app/controllers/vacancies_controller.rb

```diff
@@ -40,6 +40,7 @@ def params
 
   def algolia_search_params
     strip_empty_checkboxes(:jobs_search_form, %i[job_roles phases working_patterns])
+    params['job_roles'] = [params['job_roles']] if params['job_roles'].is_a?(String)
```

Comment:
Why is this necessary?

Reply:
When clicking on a job role facet link the value of "job_roles' in the parameters is a string. As string are not permitted for job_roles. We need to convert it to an array I believe. Perhaps there's a better way of doing this? 
```
.permit(:keyword, :location, :location_category, :radius, :jobs_sort, :page,
              job_roles: [], phases: [], working_patterns: [])
```

Reply:
can you add `:job_roles` as an argument in the permit?